app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (

```

Comment:
Do not forget to place one blank line between the different groups of imports according to PEP-8.